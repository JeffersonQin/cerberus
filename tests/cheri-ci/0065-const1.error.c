const int * volatile x;

int main(void)
{
  x = 0;  // this is ok
  *x = 0; // this is an error
}
