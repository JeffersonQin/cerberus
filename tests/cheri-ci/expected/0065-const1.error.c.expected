cheri-ci/0065-const1.error.c:6:3: error: constraint violation: expression is not assignable
  *x = 0; // this is an error
  ~~ ^
ยง6.5.16#2: 
2   An assignment operator shall have a modifiable lvalue as its left operand.
